app/code/Magento/Ui/view/base/web/js/form/element/abstract.js

```diff
@@ -119,7 +119,7 @@ define([
             this._super();
 
             scope = this.dataScope;
-            name = scope.split('.').slice(1);
+            name = scope.split('.').length > 1 ? scope.split('.').slice(1) : scope.split('.');
```

Comment:
I think would be great to extract scope.split('.') to variable in order not to calculate it 3 times